.travis.yml

```diff
@@ -128,42 +81,31 @@ before_script:
   - tensorboard/tools/license_test.sh
   # Make sure that IPython notebooks have valid Markdown.
   - tensorboard/tools/docs_list_format_test.sh
+  - elapsed "before_script (done)"
 
 # Commands in this section should only fail if it's our fault. Travis will
 # categorize them as 'failed', rather than 'error' for other sections.
 script:
+  - elapsed "script"
   # Note: bazel test implies fetch+build, but this gives us timing.
-  - bazel fetch //tensorboard/...
-  - bazel build //tensorboard/...
+  - elapsed && bazel fetch //tensorboard/...
+  - elapsed && bazel build //tensorboard/...
   - |
-    # When TensorFlow is not installed, run a restricted subset of tests.
-    if [ -z "${TF_VERSION_ID}" ]; then
-      test_tag_filters=support_notf
+    # Run tests (only a restricted subset if TensorFlow is not installed).
```

Comment:
Ah ok, so that's why it's structured like this.  I reverted the change but moved the variable setting up to `before_script` so that if that part fails the build will error, rather than proceeding with the test command.